docs/guides/install-qiskit.mdx

```diff
@@ -26,7 +26,11 @@ Whether you will work locally or in a cloud environment, the first step for all
 
 1. Install Python. Check the "Programming Language" section on the [Qiskit PyPI project page](https://pypi.org/project/qiskit/) to determine which Python versions are supported by the most recent release. For download instructions, see the [Python Beginners Guide.](https://wiki.python.org/moin/BeginnersGuide/Download)
 
-    We recommend that you use [Python virtual environments](https://docs.python.org/3.10/tutorial/venv.html) to separate Qiskit from other applications.
+    It is recommended that you use [Python virtual environments](https://docs.python.org/3.10/tutorial/venv.html) to separate Qiskit from other applications.
+
+    <Admonition type = "note">
+    These instructions use the standard Python distribution from pypi.org. However, you can use other Python distributions, such as [Anaconda](https://docs.anaconda.com/anaconda/) or [miniconda](https://docs.anaconda.com/miniconda/), along with other dependency management workflows like [Poetry](https://python-poetry.org/docs/).
```

Comment:
Not sure if it's useful but should we hyperlink pypi.org?

Reply:
sure